components/src/core/ChannelValue/ChannelValue.tsx

```diff
@@ -1,17 +1,29 @@
-import React, { useCallback, HTMLAttributes } from 'react';
+import React, { useCallback } from 'react';
 import Typography from '@mui/material/Typography';
-import makeStyles from '@mui/styles/makeStyles';
-import createStyles from '@mui/styles/createStyles';
-import clsx from 'clsx';
+import { cx } from '@emotion/css';
 import PropTypes from 'prop-types';
+import { Box, BoxProps } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import channelValueClasses, { ChannelValueClasses, getChannelValueUtilityClass } from './ChannelValueClasses';
+import { unstable_composeClasses as composeClasses } from '@mui/base';
 
-export type ChannelValueClasses = {
-    root?: string;
-    icon?: string;
-    units?: string;
-    value?: string;
+const useUtilityClasses = (ownerState: ChannelValueProps): any => {
```

Comment:
Here type should be **_Record<"text" | "prefix" | "root" | "icon" | "suffix" | "value" | "units", string>_** instead of any

- [ ] Please replace type any

Reply:
good catch!